tests/unit/assessment/test_assessment.py

```diff
@@ -1,17 +1,23 @@
-from databricks.labs.ucx.assessment.crawlers import spark_version_compatibility
+from databricks.labs.ucx.assessment.crawlers import (
+    DBRVersion,
+    DBRVersionInfo,
+    parse_spark_version,
+)
 
 
 def test_spark_version_compatibility():
-    assert "unsupported" == spark_version_compatibility("custom:snapshot__14")
```

Comment:
Revert

Reply:
I've removed class-level stuff, but the new implementation is way cleaner than just returning strings - it's well typed and provides convenient interface to operate with versions.

Reply:
revert. it doesn't belong to this PR.